waffle-compiler/test/compiler/wrappers/compileNative.ts

```diff
@@ -19,6 +19,16 @@ describe('UNIT: compileNative', () => {
     expect(actualCommand).to.match(new RegExp(expectedCommand));
   });
 
+  it('buildCommand with custom compiler', async () => {
```

Comment:
I couldn't actually execute the test suite as it fails with various issues on macOS. One being the lack of docker.